packages/react-native-reanimated/src/NativeReanimated/NativeReanimated.ts

```diff
@@ -78,19 +78,23 @@ See \`https://docs.swmansion.com/react-native-reanimated/docs/guides/troubleshoo
 }
 
 export class NativeReanimated {
-  private InnerNativeModule: NativeReanimatedModule;
+  #nativeWorklets: NativeWorklets;
+  #nativeReanimatedModule: NativeReanimatedModule;
 
   constructor() {
+    const nativeWorklets = new NativeWorklets();
     // These checks have to split since version checking depend on the execution order
     if (__DEV__) {
       assertSingleReanimatedInstance();
     }
     global._REANIMATED_VERSION_JS = jsVersion;
     if (global.__reanimatedModuleProxy === undefined) {
-      const valueUnpackerCode = getValueUnpackerCode();
-      ReanimatedModule?.installTurboModule(valueUnpackerCode);
+      ReanimatedModule?.installTurboModule();
     }
-    if (global.__reanimatedModuleProxy === undefined) {
+    if (
+      global.__workletsModuleProxy === undefined ||
```

Comment:
I think this part of the check should be moved as a method to the `NativeWorklets` class.

Reply:
It actually is there, I forgor to remove it.